includes/gateways/stripe/includes/admin/class-give-stripe-admin-settings.php

```diff
@@ -686,95 +703,198 @@ public function stripe_modal_checkout_status( $status = 'enabled' ) {
-							</p>
-						<?php endif; ?>
-					<?php endif; ?>
-					<?php
-					if ( ! defined( 'GIVE_STRIPE_VERSION' ) ) {
-						?>
-						<p class="give-field-description">
+						</div>
+						<div class="give-stripe-account-manager-list">
 							<?php
-							echo sprintf(
-								__( 'The free Stripe payment gateway includes an additional 2%% fee for processing one-time donations. This fee is removed by activating the premium <a href="%1$s" target="_blank">Stripe add-on</a> and never applies to subscription donations made through the <a href="%2$s" target="_blank">Recurring Donations add-on</a>. <a href="%3$s" target="_blank">Learn More ></a>', 'give' ),
-								esc_url( 'https://givewp.com/addons/stripe-gateway/' ),
-								esc_url( 'https://givewp.com/addons/recurring-donations/' ),
-								esc_url( 'http://docs.givewp.com/addon-stripe' )
-							);
-							?>
-						</p>
-						<?php
-					}
-					?>
+							if ( is_array( $stripe_accounts ) && count( $stripe_accounts ) > 0 ) {
+								foreach ( $stripe_accounts as $slug => $details ) {
+									$account_name       = ! empty( $details['account_name'] ) ? $details['account_name'] : give_stripe_convert_slug_to_title( $slug );
+									$account_email      = ! empty( $details['account_email'] ) ? $details['account_email'] : '';
+									$account_country    = ! empty( $details['account_country'] ) ? $details['account_country'] : '';
+									$stripe_account_id  = ! empty( $details['give_stripe_user_id'] ) ? $details['give_stripe_user_id'] : '';
+									$disconnect_message = ( 'connect' === $details['type'] ) ?
+										sprintf(
+											esc_html__( 'Are you sure you want to disconnect GiveWP from Stripe? If disconnected, this website and any others sharing the same Stripe account (%1$s) that are connected to GiveWP will need to reconnect in order to process payments.', 'give' ),
+											$stripe_account_id
+										) :
+										esc_html__( 'Are you sure you want to disconnect GiveWP from Stripe?', 'give' );
+									$disconnect_url     = ( 'connect' === $details['type'] ) ?
+										give_stripe_disconnect_url( $stripe_account_id, $slug ) :
+										add_query_arg(
+											[
+												'post_type' => 'give_forms',
+												'page'    => 'give-settings',
+												'tab'     => 'gateways',
+												'section' => 'stripe-settings',
+												'give_action' => 'disconnect_manual_stripe_account',
+												'account' => $slug,
+											],
+											admin_url( 'edit.php' )
+										);
+
+									?>
+									<div id="give-stripe-<?php echo $slug; ?>" class="give-stripe-account-manager-list-item">
+										<div class="give-stripe-account-name-wrap">
+											<span class="give-stripe-account-name">
+												<?php _e( $account_name, 'give' ); ?>
+											</span>
+											<span class="give-field-description give-stripe-account-email">
+												<?php _e( $account_email, 'give' ); ?>
```

Comment:
@mehul0810 translation function does not work with dynamic string. can we is this?